core/app/models/spree/variant.rb

```diff
@@ -102,10 +102,14 @@ class Variant < Spree::Base
 
     self.whitelisted_ransackable_associations = %w[option_values product prices default_price]
     self.whitelisted_ransackable_attributes = %w[weight sku]
-    self.whitelisted_ransackable_scopes = %i(product_name_or_sku_cont)
+    self.whitelisted_ransackable_scopes = %i(search_by_product_name_or_sku)
 
-    def self.product_name_or_sku_cont(query)
```

Comment:
@aplegatt please don't remove the previous method, we need them both for compatibility issues